project-example/week-9/eventonica/eventonica-postgres.md

```diff
@@ -65,94 +65,112 @@ In addition to the usual steps:
 
    ```bash
-   npm install pg-promise
+   npm install pg
    ```
 
-1. Copy the setup instructions for `pg-promise` in your db folder (you have to create one). Your connection string is probably something like `postgres://localhost:5432/eventonica`. You should not need a username or password if you [setup postgres](../../../databases/installing-postgresql.md) correctly.
+1. Copy the setup instructions for [pg](https://node-postgres.com/api/pool) in your db folder (you have to create one). Your connection string is probably something like `postgres://localhost:5432/eventonica`. You should not need a username or password if you [setup postgres](../../../databases/installing-postgresql.md) correctly.
 
    ```javascript
    // server/db/db-connection.js;
-   import pgPromise from 'pg-promise';
-
-   // Create Database Connection
-   const pgp = pgPromise({});
-
-   const db = pgp('postgres://localhost:5432/eventonica');
+   import pg from 'pg';
+   const { Pool } = pg;
+   const db = new Pool({
+     connectionString: 'postgres://localhost:5432/eventonica'
+   });
 
    export default db;
    ```
 
 1. Update your Eventonica methods (addUser, delete etc) to use SQL commands.
 
    - Use `psql` or `PGAdmin` to test your SQL commands.
-   - Add them to your JS using the package `pg-promise` - you can find example queries [here](https://github.com/vitaly-t/pg-promise/wiki/Learn-by-Example).
-   - Note that `pg-promise` requires you to specify how many rows, if any, a query should return. For example, `db.any` indicates that the query can return any number of rows, `db.one` indicates that the query should return a single row, and `db.none` indicates that the query must return nothing.
+   - Add them to your JS using the package `pg` - you can find example queries [here](https://node-postgres.com).
 
-   A GET request will be your first endpoint. You can put raw SQL that will touch the API database inside the db.any(). All users will be get by using `SELECT` clause.
+   A GET request will be your first endpoint. You can put raw SQL that will touch the API database inside the db.query(). All users will be get by using `SELECT` clause.
 
    ```js
-   // server/routes/ users.mjs;
-   ......
-   import db from "../db/db-connection.js";
-
+   // server/index.js;
+    ........
+   import db from './db/db-connection.js';
+    ........
    /* GET users listing. */
-
-   router.get('/', async function (req, res, next) {
-
+   app.get('/api/users', async (req, res) => {
      try {
-       const users = await db.any('SELECT * FROM users', [true]);
+       const users = await db.query('SELECT * FROM users');
        res.send(users);
      } catch (e) {
        return res.status(400).json({ e });
      }
    });
 
-   /* post request goes here */
-
-   /* delete request goes here  */
+      <!-- POST request goes here -->
+      <!-- PUT request goes here -->
+      <!-- DELETE request goes here -->
```

Comment:
Might want to switch these back to javascript comments rather than html comments.